src/Layer/ElevationLayer.js

```diff
@@ -7,6 +7,8 @@ import { RasterElevationTile } from 'Renderer/RasterTile';
  * an ElevationLayer. Default is true. You should not change this, as it is used
  * internally for optimisation.
  * @property {number} noDataValue - Used to specify a **null** or **no data value** in the elevation terrain.
+ * @property {number} zmin - Used to specify a minimum value under which the elevation terrain will be clamped.
+ * @property {number} zmax - Used to specify a maximum value over which the elevation terrain will be clamped.
```

Comment:
I'm wondering if we shouldn't rename these properties to something like `minClampValue` and `maxClampValues` to avoid confusions with actual `zmin` or `zmax` of the layer and to be closer to the `config` object passed to the constructor . WDYT @ftoromanoff @mgermerie ?

Reply:
For me zmin and zmax are the min and max elevation for a layer after clamping, to avoid absurd values in the data.
So it's indeed the zmin and zmax of a layer, but these can be played with throught the clampValues parameters.

Reply:
It is the case when the values are actually clamped but if the min clamp value is lower than the min value of the layer it is not the case for instance but this might be a detail. My remark was also to be more consistent with the configuration value. But no strong opinions on this, I will let you decide.

Reply:
the clamp parameters values are only used if there is a data value lower (or higher) that the clamp value on a tile. It will not be the default value taken. (Zmin and zmax are a 'global' value for the layer, but bounding boxes will be calculated on each tile.

Reply:
Yes that's what I understood too :slightly_smiling_face: let's keep these names